Documentation/RunCommand.md

```diff
@@ -0,0 +1,24 @@
+Run Command Tool
+===========================
+The Run Command Tool has a published contract of inputs and outputs to encapsulate the dev workflow. It parses arguments and maps properties to tools in order to run the command given by the user. It also provides documentation of the most common settings we use per repo, and the commands we can execute.
+
+The source code of the tool lives in the Build Tools repo and it is included after the Build Tools package version 1.0.26-prerelease-00601-01. 
+Every repo should have a run.cmd/sh script file that is in charge of:
```

Comment:
I add the *Italic* text in the next quote like:
>*In order to on board the Run Command Tool*, every repo should have a run.cmd/sh ....

Also I think that it would be nice if you provided a link to the run.cmd and run.sh that we have in buildtools as an example on what this scripts look like